posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -288,8 +288,12 @@ public class PostHogReplayIntegration(
         var type: String? = null
         // button inherits from textview
         if (view is TextView) {
-            // TODO: masking
-            text = view.text.toString()
+            text = if (!view.isNoCapture() && !config.maskAllInputs) {
+                // TODO: masking
+                view.text.toString()
+            } else {
+                "*".repeat(view.text.length)
```

Comment:
one thing I wish we'd done on the web was limit the length of this replaced text... sometimes we send ourselves thousands of asterisks ... would be cool to be able to `*****<truncated>` after some length - like 200 characters or something

Reply:
Not a bad idea, but if the bug is around the UI (text too big), the replay won't be useful since the original message is truncated.
Maybe `*****<truncated - 200 more chars>` :D